src/App/Main/Product/ProductListItem.js

```diff
@@ -2,6 +2,8 @@ import { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
+import { generatePath } from 'react-router';
```

Comment:
Should be imported from `react-router-dom`. `react-router-dom` is default package that reexports all stuff from `react-router` and contains DOM bindings